### DISCUSSION TITLE
Volo.Abp.Account.Pro.Public.Application - AccountAppService class

### DISCUSSION THREAD

**Discussion by spospisil** [abpframework/abp#13798] [Q&A] [answered]
Hi,

There are two public methods (SendPhoneNumberConfirmationTokenAsync and  GetConfirmationStateAsync) in this class that are not marked as 'virtual' resulting in developers not being able to inherit or override them according to the ABP Best Practices link ([https://docs.abp.io/en/abp/latest/Best-Practices/Application-Services) for Application Services.

Could this be addressed/fixed?

Steve

**Comment by maliming** [comment, marked as answer]
ok, I will fix that.
